api/src/main/java/com/okta/idx/sdk/api/client/BaseIDXClient.java

```diff
@@ -66,7 +66,7 @@
 
 final class BaseIDXClient implements IDXClient {
 
-    private static final String USER_AGENT_HEADER_VALUE = "okta-idx-java/1.0.0";
+    private static final String USER_AGENT_HEADER_VALUE = "okta-idx-java/2.0.0";
```

Comment:
Can we generate this in the build scripts so we don't have to remember to update it?

Reply:
agreed, will create a PR for that.